src/components/scans/scans.js

```diff
@@ -81,13 +81,14 @@ class Scans extends React.Component {
     return (
       <div className="form-group">
         <Tooltip key="mergeButtonTip" content="Merge selected scan results into a single report">
-          <ButtonPf3
+          <Button
             id="merge-reports"
-            disabled={viewOptions.selectedItems.length <= 1}
+            variant={ButtonVariant.secondary}
```

Comment:
required:
investigate with UX @bclarhk to see if this should be a primary button